core/src/main/scala/org/apache/spark/util/Utils.scala

```diff
@@ -2599,14 +2599,10 @@ private[spark] object Utils extends Logging {
 
 private[util] object CallerContext extends Logging {
   val callerContextSupported: Boolean = {
-    SparkHadoopUtil.get.conf.getBoolean("hadoop.caller.context.enabled", false) && {
+    SparkHadoopUtil.get.conf.getBoolean("hadoop.caller.context.enabled", true) && {
```

Comment:
I will change default value to `false` later. Just to test if this PR fixes the issue on Jenkins.